webpack/common.js

```diff
@@ -167,24 +167,34 @@ const noAMDParserRule = { parser: { amd: false } };
 
 exports.noAMDParserRule = noAMDParserRule;
 
-const svgRule = {
-	test: /\.svg$/,
-	use: [
-		{
-			loader: '@svgr/webpack',
-			options: {
-				// strip width & height to allow manual override using props
-				dimensions: false,
+const svgRules = [
+	{
+		oneOf: [
+			{
+				test: /\.svg$/,
+				resourceQuery: /url/,
+				use: 'url-loader',
```

Comment:
This loader should be added as a dependency to `assets` and `storybook` workspaces